### PR TITLE
New solutions to the simulation module on USACO Guide 

### PR DESCRIPTION
**Pull request by jessechoe10**
Could you check/add these sols on USACO Guide, please?

### CODE REVIEW

**Comment by bqi343** [comment]
https://usaco.guide/general/contributing#adding-a-solution

> Then add the solution ID (usaco-690) as the last argument of the Problem constructor in the corresponding module

File: solutions/USACO dec15-speeding-ticket.mdx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+#define MAX_INT 1e18*9
+
+// Permutation
+
+#define possibilities(x) while(next_permutation(all(x))
+
+// Loops
+
+#define Trav(a,x) for (auto& a: x)
+#define For(i,a,b) for (int i = (a); i < (b); ++i)
```
Comment (thecodingwizard) [inline solutions/USACO dec15-speeding-ticket.mdx:63]: Can we remove / shorten these templates, or make the consistent with https://usaco.guide/general/code-con#c ?

File: solutions/USACO dec15-speeding-ticket.mdx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+	For(i, 0, N){
+
+	int length, speed; cin >> length >> speed;
+
+	For(j, start, start+length){
+		limit[j] = speed;
+	}
+
+	start += length;
+
```
Comment (thecodingwizard) [inline solutions/USACO dec15-speeding-ticket.mdx:92]: indentation?

File: solutions/USACO dec17-milk-measurement.mdx [hunk 3]
```diff
@@ -0,0 +1,141 @@
+---
+id: usaco-592
+title: USACO December 2017 Bronze - The Bovine Shuffle
+author: Jesse Choe
+---
+
+[Official Analysis]: http://www.usaco.org/current/data/sol_measurement_bronze_dec17.html
```
Comment (thecodingwizard) [inline solutions/USACO dec17-milk-measurement.mdx:7]: There's already an official C++ implementation; is there a particular reason to include another one?

File: solutions/USACO dec17-the-bovine-shuffle.mdx [hunk 4]
```diff
@@ -0,0 +1,119 @@
+---
+id: usaco-760
+title: USACO December 2017 Bronze - The Bovine Shuffle
+author: Jesse Choe
+---
+
+[Official Analysis]: http://www.usaco.org/current/data/sol_shuffle_bronze_dec17.html
```
Comment (thecodingwizard) [inline solutions/USACO dec17-the-bovine-shuffle.mdx:7]: there's already an official C++ implementation...

File: solutions/USACO open17-thelostcow.mdx [hunk 5]
```diff
@@ -0,0 +1,124 @@
+---
+id: usaco-735
+title: USACO 2017 US Open Bronze - The Lost Cow
+author: Jesse Choe
+---
+
+[Official Analysis]: http://www.usaco.org/current/data/sol_lostcow_bronze_open17.html
```
Comment (thecodingwizard) [inline solutions/USACO open17-thelostcow.mdx:7]: There's already a C++ implementation...

File: solutions/USACO open18-team-tic-tac-toe.mdx [hunk 6]
```diff
@@ -0,0 +1,167 @@
+---
+id: usaco-831
+title: USACO US Open 2018 - Team Tic Tac Toe
+author: Jesse Choe
+---
+
+[Official Analysis]: http://www.usaco.org/current/data/sol_tttt_bronze_open18.html
```
Comment (thecodingwizard) [inline solutions/USACO open18-team-tic-tac-toe.mdx:7]: there's already C++ impl...

**Review by thecodingwizard — CHANGES_REQUESTED** [review]

**Comment by thecodingwizard** [comment]
Thanks for the PR @jessechoe10 !

For the problems that already have official C++ implementations, we generally aren't looking for alternative C++ implementations (unless the alternative implementation takes a different approach or is written better / differently from the official implementation).

Also, would it be possible to fix some of the indentation + make the template consistent with the rest of the guide for dec15-speeding ticket?

Thanks!

File: solutions/USACO dec15-speeding-ticket.mdx (hunk 1 above)
Comment (bqi343) [reply solutions/USACO dec15-speeding-ticket.mdx:63]: I think this is fine :P

File: solutions/USACO open18-team-tic-tac-toe.mdx (hunk 6 above)
Comment (jessechoe10) [reply solutions/USACO open18-team-tic-tac-toe.mdx:7]: Does it use sets and maps, though?

Comment (thecodingwizard) [reply solutions/USACO open18-team-tic-tac-toe.mdx:7]: oops mb, you're right. Can we add a quick note after this line that's like "official solution doesn't use sets/maps, here's one that does"?

Comment (jessechoe10) [reply solutions/USACO open18-team-tic-tac-toe.mdx:7]: yep ofc. idk if that was directed to @bqi343

**Review by bqi343 — APPROVED** [review]
Left `tttt` and `speeding` in.

**Pull request merged**